acquia/acquia_cms
# ACMS-1532: Removed non-required core patch.

**Motivation**
Fixes #ACMS-1531

**Proposed changes**
Remove non-required core patch.

**Alternatives considered**
NA

**Testing steps**
- composer create-project acquia/drupal-recommended-project project
- cd project
- composer require drupal/core:~9.5.0
- composer require acquia/acquia_cms
- verify no error while patching.

**Merge requirements**
